app/models/puppetdb_client.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'puppetdb'
+
+class PuppetdbClient
+  def initialize(endpoint)
+    @endpoint = endpoint
+    @client = PuppetDB::Client.new(server: @endpoint)
+  end
+
+  def ipaddress(fqdn)
+    response = @client.request('facts', ['and', ['=', 'certname', fqdn], ['=', 'name', 'ipaddress']])
+    response.data.map { |key| key['value'] }.first
```

Comment:
What should we do for machines that have more than one IP? What are the sorts of machines have more than one ipaddress fact?